api/src/services/authService.ts

```diff
@@ -111,6 +113,23 @@ class AuthService extends DatabaseService {
     });
     return await this.compareStrings(password, rows[0].password);
   }
+
+  public aTokenOptions = { expiresIn: 60 };
+  public rTokenOptions = { expiresIn: '1d' };
+  /**
+   * @description Return both access and refreshToken
+   * @param {{payload}} userObject tokens payload content
+   * @param {Reply} reply fastify instance's reply, for the use of regitered jwt plugin
+   * @returns object that contains both emitted tokens
+   */
+  public async generateTokens(
+    userObject: { id: number, pseudo: string, role: ERoles },
+    reply: Reply
+  ): Promise<{ accessToken: string, refreshToken: string }> {
+    const accessToken = await reply.jwtSign(userObject, this.aTokenOptions);
+    const refreshToken = await reply.jwtSign(userObject, this.rTokenOptions);
+    return { accessToken, refreshToken };
+  };
```

Comment:
You cannot use Request/Reply methods here, this is the job of the controller.
Services should only be aware of datas. If you want to do it this way then we need to create our own token service instead of this librairy.

Reply:
To be fair, I'm only asking that you push back this method to the controllers because I think we really should create our token handling service in a futur issue, but I prefer no solution instead of a dirty one.